include/server.h

```diff
@@ -5,11 +5,19 @@
 
 #include "connection.h"
 
+struct ServerInfo;
```

Comment:
Wait it's defined below, we shouldn't need to comment here.
It's just to make Server happy.